src/scraper/scraper.cpp

```diff
@@ -304,6 +365,8 @@ void _log(logattribute eType, const std::string& sCall, const std::string& sMess
 
     log.output(sOut);
 
+    log.closelogfile();
```

Comment:
I'm not sure I follow. Do we want to close the file each time we log?

Reply:
We have to. There is no other way to do it. The performance loss is minimal. I already checked.

Reply:
Don't we only need to close the file when we want to rotate?

Reply:
Two threads are writing to that file. If one thread closes it before rotating, the other still has an open handle, so the rename won't work.

Reply:
"Windows"